### PR TITLE
Use NOAA-GSL/EMC_post RRFS_dev branch to apply the lastest version of…

### PR DESCRIPTION
**Pull request by hu5970**

### CODE REVIEW

File: Externals.cfg [hunk 1]
```diff
@@ -38,8 +38,8 @@ required = True
 protocol = git
 repo_url = https://github.com/NOAA-GSL/EMC_post
 # Specify either a branch name or a hash but not both.
-#branch = RRFS_dev
-hash = 9b9c55c
+branch = RRFS_dev
+#hash = 9b9c55c
```
Comment (christinaholtNOAA) [inline Externals.cfg:42]: Please use a particular hash for the code that you would like to use. Using the top of a branch for workflows running in real time is hard to reconstruct for future use if we need to rebuild.

**Review by christinaholtNOAA — CHANGES_REQUESTED** [review]
Ming, please make sure that you are providing short title description, and leaving the details in the discussion box when submitting a PR.

**Comment by hu5970** [comment]
@christinaholtNOAA Changed to use hash instead of top of the branch. Thanks,

**Comment by hu5970** [comment]
@christinaholtNOAA Christina, it there further change needed for merge this? Thanks, Ming

**Review by christinaholtNOAA — APPROVED** [review]

**Pull request merged**